recipes/lxml/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "3.6.2" %}
+
+package:
+    name: lxml
+    version: {{ version }}
+
+source:
+    fn: lxml-{{ version }}.tar.gz
+    url: https://github.com/lxml/lxml/archive/lxml-{{ version }}.tar.gz
+    sha256: 90e5f33f19a8964eeb8631d477dc103c8ddbd8067a002027e3f9cc79f7fb2d80
+
+build:
+    number: 0
+    skip: True  # [win]
+    script: python setup.py install --with-xslt-config=$PREFIX/bin/xslt-config  # [not win]
+
+requirements:
+    build:
+        - python
+        - cython
+        - libxml2
+        - libxslt
+    run:
+        - python
+        - libxml2
+        - libxslt
+
+test:
+    imports:
+        - lxml
+        - lxml.etree
+        - lxml.objectify
+
+about:
+    home: http://lxml.de/
+    license: BSD 3-Clause, GPL-2.0, ZPL-2.0, and ElementTree
+    license_file: LICENSES.txt
```

Comment:
This will be a nice discussion on shifting the burden to the maintainer to help automated tools to parse licenses :grinning: (Spoilers here it will probably be useless.)